lib/rules/web/console/base/login.xyaml

```diff
@@ -15,7 +15,7 @@ login_sequence:
     if_element:
       selector: &pre_login_buttons
         # we just choose the first button per OPENSHIFTQ-12062
-        css: ".login-redhat:last-of-type, .idp:last-of-type"
+        css: ".login-redhat:first-of-type, .idp:first-of-type"
```

Comment:
It didn't work with last of type? Usually last of type is the inner-most element so must be somehow safer to use last-of-type if it works.

Reply:
Other than that LGTM.

Reply:
@akostadinov In the past, you and me made a discussion and decided to use the first-of-type. But it was changed to last-of-type by PR https://github.com/openshift/verification-tests/pull/57.  Our several cases are based on choosing the first button. @yanpzhan @yapei any reasons that change it to last-of-type?

Reply:
@yasun1 , hmm, very nice that you found the change. I don't know why it changed, whether it was discussed or just an oversight. I initially thought there was some issue with nesting elements but maybe not. So I'm ok with switching back to first-of-type. Thank you.

Reply:
@yasun1  Since the first IDP choice for OCP 4.x clusters is for kube:admin, we don't want to user `kube:admin` for testing so changed to the last one

Reply:
@yapei so the last-of-type should be a required pre-condition for OCP 4.X cluster, correct? Compared the effected cases, I think ours are much less than yours.  And for this PR has been merged, you can withdraw the first-of-type to the last-of-type in a new PR, and notice us to update our cases. Thanks.
@akostadinov maybe still need bother you merge a new PR again. :)

Reply:
Sure thing. I guess we can also select second element or third element if we wish. IIRC it was possible but needs to be checked.